app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
```suggestion
class NotWearingMaskError(Exception):
```

`
The built-in exception classes can be subclassed to define new exceptions; programmers are encouraged to derive new exceptions from the Exception class or one of its subclasses, and not from BaseException.
`
Source: [Link](https://docs.python.org/3/library/exceptions.html)